store/types.go

```diff
@@ -0,0 +1,43 @@
+package store
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	"math/big"
+)
+
+/* Account */
+type Account struct {
+	Balance *big.Int          // total amount avaliable to be spent
+	Unspent []plasma.Position // position of unspent transaction outputs
+	Spent   []plasma.Position // position of spent transaction outputs
+}
+
+/* Wrap plasma deposit with spend information */
+type Deposit struct {
+	Deposit plasma.Deposit
+	Spent   bool
+	Spender []byte
```

Comment:
Both deposit and Output can have multiple spenders? Are you just going to append to single byte array

Reply:
Spender is TxHash that spent it

Reply:
Ok looking at this, I thought it was the Spender address. Maybe make it clear with `SpenderTx` and an inline comment?